src/model/modifier/__tests__/AtomicBlockUtils-test.js

```diff
@@ -38,6 +38,7 @@ const assertAtomic = state => {
         key: block.getKey(),
         type: block.getType(),
```

Comment:
Now that all transactions are completed we should be able to include some basic tests for the tree data structure on the AtomicBlockUtils-test file